docs/evm/how-it-works.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Differences How EVM on Flow Works
+sidebar_label: How it Works
+sidebar_position: 2
+---
+
+# Differences between EVM on Flow and Ethereum
```

Comment:
Would update this as well to: How EVM on Flow Works

Reply:
fixed